build: promotion fails because version file is missing

Saw this today while promoting v0.4.0 to alpha channel:
```
> make promote CHANNEL=alpha VERSION=v0.4.0
promoting release v0.4.0 to channel alpha
uploading version to S3 bucket rook-release

The user-provided path release/version does not exist.
build/makelib/release.mk:111: recipe for target 'release.promote' failed
make[1]: *** [release.promote] Error 255
Makefile:251: recipe for target 'promote' failed
make: *** [promote] Error 2
```

Fixed in place manually to continue the promotion successfully.

--- a/build/release/release.sh
+++ b/build/release/release.sh
@@ -33,6 +33,7 @@ case ${action} in
         fi
 
         echo promoting release ${RELEASE_VERSION} to channel ${RELEASE_CHANNEL}
+        write_version_file
         publish_version_file
         github_create_or_replace_release
         ;;

